src/pages/home.tsx

```diff
@@ -1,9 +1,69 @@
-import React from "react";
+import '../components/home/home.css';
+import { ReactComponent as PageLinksIcon } from '../assets/home/arrow.svg';
+import { ReactComponent as CreateHackathonIcon } from '../assets/home/createHackathonIcon.svg';
+import { ReactComponent as HackathonAnalyticsIcon } from '../assets/home/hackathonAnalyticsIcon.svg';
+import { Link } from 'react-router-dom';
 
 export default function Home() {
-  return (
-    <div>
-      <h1>HOME</h1>
-    </div>
-  );
+	// TODO: This user name should be generic.
+	let userName: string = 'John';
+
+	const pageLinks = [
+		{ route: '/hackathon', buttonText: 'Go to Hackathon' },
+		{ route: '/attendees', buttonText: 'Attendees' },
+		{ route: '/sponsors', buttonText: 'Sponsors' },
+		{ route: '/edit-details', buttonText: 'Edit Details' },
+		{ route: '/schedule', buttonText: 'Schedule' },
+		{ route: '/settings', buttonText: 'Settings' },
+	];
+
+	return (
+		<div id="page">
+			<div id="page-header">
+				<div id="page-header-main">
+					<div id="page-header-main-generic-profile-icon" />
+					<h1 id="page-header-main-text">Welcome, {userName}.</h1>
+				</div>
+
+				<Link to="/analytics">
+					<button id="page-header-button">
+						<HackathonAnalyticsIcon />
+						<p>Hackathon Analytics</p>
+					</button>
+				</Link>
+			</div>
+
+			<h1 className="page-subheader-text" id="page-subheader-1-text">
+				Upcoming Hackathon
+			</h1>
+
+			<div id="page-links">
+				<h1 id="page-links-header">Fall 2022</h1>
+
+				<div id="page-links-grid">
+					{pageLinks.map(({route, buttonText}) => (
+						<Link key={route} to={route}>
+							<button className="page-links-grid-button">
+								<PageLinksIcon />
+								<p>{buttonText}</p>
```

Comment:
Seems like the icon for each link, when at the largest screen size, can overlap with the text. I'm viewing this on a giant monitor, but when I make the browser window smaller at the same breakpoint they separate. I suspect a laptop would be small enough you wouldn't see this, but even if you zoom out enough on a smaller screen you can see this issue.

As soon as we get to the smaller breakpoint and they are in two columns, it's all fine and dandy.

<img width="1728" alt="image" src="https://user-images.githubusercontent.com/55410292/194720686-f3fba3fa-bd10-4076-b6c0-649745b9b222.png">

<img width="1315" alt="image" src="https://user-images.githubusercontent.com/55410292/194720698-79bd8ae3-eb94-4249-9fb4-c43a5bc8b315.png">

<img width="904" alt="image" src="https://user-images.githubusercontent.com/55410292/194720705-770d50d0-3ba7-41c8-a0c6-bc52986287b9.png">